packages/devtools_app/lib/src/analytics/prompt.dart

```diff
@@ -107,7 +105,8 @@ class _AnalyticsPromptState extends State<AnalyticsPrompt> {
           ),
           TextSpan(
             text: 'privacy policy',
-            style: textTheme.subtitle1.copyWith(color: const Color(0xFF54C1EF)),
+            style:
+                textTheme.subtitle1!.copyWith(color: const Color(0xFF54C1EF)),
```

Comment:
Do we want to provide default in case `textTheme.subtitle1` is null instead of null checking?

Reply:
added ?